tests/feature/test_outline_empty_values.py

```diff
@@ -62,3 +62,65 @@ def test_outline():
     result = pytester.runpytest("-s")
     result.assert_outcomes(passed=1)
     assert collect_dumped_objects(result) == ["#", "", ""]
+
+
+def test_scenario_with_empty_example_values_none_transformer(pytester):
```

Comment:
please add a docstring explaining what's the issue we are covering here

Reply:
Good point - I'll also add an example in the README as I realised this is probably the helpful place for people wanting to know how to do it